libs/no_asan/no_asan.cpp

```diff
@@ -0,0 +1,416 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Includes all required ASAN externs to link binaries that include ASAN instramented binaries
+ * but don't enable ASAN themselves.
+ * These binaries are not expected to work unless they are also compiled with ASAN
+ * and link the proper ASAN libs.
+ * For more information, see /docs/AddressSanitizing.md within this repository.
+ **/
+
+// TODO: Split asan and coverage into different libs
```

Comment:
Is there a github issue filed whose number could be added into this comment?

Reply:
This is just a leftover comment, will remove on the next commit.